app/handlers/signup_verify_email.rb

```diff
@@ -15,12 +15,7 @@ def authorized?
   end
 
   def handle
-    run(ConfirmByPin, contact_info: options[:pre_auth_state], pin: pin_params.pin)
-    # lms students do not receive passwords so the account needs to be created now
-    if options[:pre_auth_state].signed_student?
-      run(SignupExternalStudent, pre_auth_state: options[:pre_auth_state], already_verified: true)
-      options[:session].sign_in!(outputs.user)
```

Comment:
Tutor choosing to not use the external student stuff doesn't necessarily mean we should get rid of it.  I know I agreed with the opposite earlier, but having second thoughts.  If we have code that works and tests for it and an expectation that it will be used in the near term, should we consider keeping it?

Reply:
In the final iteration of this refactor, which is currently on dev, you can direct a student directly to the student signup form.
https://dev.openstax.org/accounts/signup/start?role=student

Perhaps this is slightly different than what this code is doing though? Or just need to have it accept an email param to prefill the student signup email for ease?

Reply:
the LMS sends a signed email address and name to Accounts, which we trust without needing the student to reverify it.  Leads to an accelerate Account creation, which is a nice benefit for users using an LMS.  That's what this was doing.  

But, as Dante said, Tutor stopped using this because of a Tutor bug, so we could kill it and just remember to come back for it later.  You can resolve this if you want to.

Reply:
I'm going to resolve because if we want to do this again, I think we'll need to rearchitect the way we are passing the information around. It also looks like we are more interested in moving towards the iframe login/signup over signed params from conversations with Tom.